test/upgrade/create-in-current_source-kafka-sink.td

```diff
@@ -15,8 +15,3 @@
 > CREATE SINK upgrade_kafka_sink FROM static_view
   INTO KAFKA CONNECTION kafka_conn TOPIC 'upgrade-kafka-sink'
   FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY '${testdrive.schema-registry-url}'
-
```

Comment:
These messages are checked in the corresponding `check` test.

Normally `verify` consumes the messages from the topic, so it's not totally obvious to me why this used to work...